src/core/types/base/mempool.ts

```diff
@@ -60,8 +61,11 @@ export function showTxMemory() {
  *@param mempool The mempool(state) to process.
  *@return An array of swap, send and swap-operation messages that exist in the `mempool`.
  */
-export function processMempool(mempool: Mempool): Array<MempoolTrade> {
-	const mempoolTrades = [];
+export function processMempool(
+	mempool: Mempool,
+	ignoreAddresses: Set<string>,
+): [Array<MempoolTrade>, Array<{ sender: string; reciever: string }>] {
```

Comment:
why would we return the sender again in the seond index objects? we have it in MempoolTrade already?